dask/array/core.py

```diff
@@ -1219,19 +1245,20 @@ def __array__(self, dtype=None, **kwargs):
         return x
 
     def __array_function__(self, func, types, args, kwargs):
-        import dask.array as module
+        if func not in _HANDLED_FUNCTIONS:
+            warnings.warn(
+                "`{}` is not implemented by dask, explicitly "
```

Comment:
Maybe include the module name in the error message, too, e.g., `func.__module__ + '.' + func.__name__`.

Reply:
done